test/test-thread-affinity.c

```diff
@@ -0,0 +1,63 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
+ */
+
+#include "uv.h"
+#include "task.h"
+
+#include <string.h>
+
+static void check_affinity(void* arg) {
+  int r;
+  char *cpumask = arg;
+  int cpumasksize = uv_thread_cpumasksize();
+
+  uv_thread_t tid = uv_thread_self();
+  uv_thread_setaffinity(&tid, cpumask, NULL, cpumasksize);
+  r = uv_thread_setaffinity(&tid, cpumask + cpumasksize, cpumask, cpumasksize);
+  if (r != 0)
+    cpumask[0] = cpumask[1] = -1;
+}
+
+TEST_IMPL(thread_affinity) {
+  int t1first, t1second, t2first, t2second;
+  int cpumasksize;
+  char *cpumask;
+  uv_thread_t threads[2];
+
+  cpumasksize = uv_thread_cpumasksize();
+  t1first = cpumasksize * 0;
+  t1second = cpumasksize * 1;
+  t2first = cpumasksize * 2;
+  t2second = cpumasksize * 3;
+
+  cpumask = (char*)calloc(4 * cpumasksize, sizeof(char));
```

Comment:
`sizeof(char) == 1`.  I'd just use a literal 1 because `sizeof(char)` always looks odd to me.